Makefile

```diff
@@ -254,7 +259,10 @@ endif
 
 # We are currently filtering out a few packages from bulk builds
 # since they are not getting published in Docker HUB
-PKGS=$(shell ls -d pkg/* | grep -Ev "eve|test-microsvcs")
+PKGS_$(ZARCH)=$(shell ls -d pkg/* | grep -Ev "eve|test-microsvcs")
+PKGS_riscv64=pkg/alpine pkg/ipxe pkg/mkconf pkg/mkimage-iso-efi pkg/grub     \
```

Comment:
Here you're just clobbering the output of the line below. So some XXX or TBD cleanup comment might make sense.

Reply:
That's actually an idiomatic Makefile sequence of how one does case'ing based on literal values of the vars. Like, it isn't some kind of a hack or exception it is basically a legitimate way to do a "default" section in the usual case statement.